proposal/color-4-new-spaces.md

```diff
@@ -1683,14 +1758,27 @@ This function is also available as a global function named `adjust-color()`.
     > This allows e.g. `color.adjust($color, $channel1: 0.25)` for adjusting
     > color channels in unknown color spaces.
 
-  * If `key` is not the name or index of a channel in `channels`, throw an
-    error.
+  * If `key` is not the name or index of a channel in `channels`:
 
-  * Set `adjust` to the result of [validating] `adjust` as `key` channel in
-    `space`.
+    * If `$space` is specified, throw an error.
+
+    * If `color` is not a [legacy color], throw an error.
+
+    * If `key` is one of `red`, `green`, or `blue`:
+
+      * Let `legacy-color` be the result of [converting] `color` to `rgb`.
+
+    * Otherwise, if `key` is one of `hue`, `saturation`, or `lightness`:
+
+      * Let `legacy-color` be the result of [converting] `color` to `hsl`.
+
+    * Otherwise, if `key` is one of `whiteness`, or `blackness`:
+
+      * Let `legacy-color` be the result of [converting] `color` to `hwb`.
+
+    * Otherwise, throw an error.
 
-    > This ensures that `adjust` is percent-converted when necessary, and
-    > throws errors when the adjustment is not a valid value for the space.
+    * Set `channels` to be a list of `legacy-color`'s channels.
 
   * Let `channel` be the channel in `channels` with name or index of `key`.
```

Comment:
I think this should be "the value of the channel", so distinguish it from the channel's metadata.